### DISCUSSION TITLE
Strange error

### DISCUSSION THREAD

**Discussion by SenorCarlito1** [community/community#111371] [New to GitHub]
I feel so dumb for not being able to figure this out but I have been trying to connect my repository to azure and it worked just fine but I cannot use my github desktop to push or pull, I can only do so with command line and I get this strange error whenever I fetch from the origin using the desktop. I have been trying to find solutions and I keep going in circles, I cannot seem to find the solution to this issue.

![image](https://github.com/community/community/assets/118745644/ab0c9503-2dc3-444a-970c-7cff0a29084e)

**Comment by amankr1469** [comment]
Go to Control Panel -> Credential Manager -> Windows Credentials. There add your Github username and PAT in 'password'. 
You can get PAT from the Developers setting in your GitHub account.

**Comment by queenofcorgis** [comment]
@SenorCarlito1 if @amankr1469 response helped you out, please `Mark as answer`, this will highlight the comment and help other community members quickly find the answer as well.
